README.md

```diff
@@ -1,11 +1,9 @@
-# Zulip Google Code-In Submissions Repository
-
-This is a repository for student submissions in Zulip's Google Code In, 
+https://github.com/zulip/zulip-gci/blob/master/tasks/2017/submit-a-pull-request.md repository for student submissions in Zulip's Google Code In, 
```

Comment:
What's the story with this change?

Reply:
Accidentally changed that. @timabbott Kindly review now.